### PR TITLE
Address some version compatibility issues with the OpenSeadragon library.

### PR DESCRIPTION
**Pull request by adam-vessey**
Older versions build the paths to the navigation button images (zoom/home/fullscreen buttons) differently, and version numbers do not suffice to differentiate how to provide the "prefixUrl"... Instead, we override all the image URLs with how the current code provides them.

Addresses OnTime 904.

### CODE REVIEW

File: islandora_openseadragon.module [hunk 1]
```diff
@@ -74,14 +74,43 @@ function islandora_openseadragon_callback($params = NULL, $fedora_object = NULL)
...
   // Variable fedora_object included in results, but not used.
   $library_path = libraries_get_path('openseadragon');
   $module_path = drupal_get_path('module', 'islandora_openseadragon');
   $variables['viewer_id'] = 'islandora-openseadragon';
   $settings = array(
     'id' => $variables['viewer_id'],
-    'prefixUrl' => url($library_path),
+    // XXX: Can't use url(), as it can add in language prefixes, which will break
+    // the URL.
+    'prefixUrl' => "$base_url/$library_path/images/",
```
Comment (nigelgbanks) [inline islandora_openseadragon.module]: I think http://api.drupal.org/api/drupal/includes!file.inc/function/file_create_url/7. Might be what your looking for.

**Comment by nigelgbanks** [comment]
I've tested it and it seems to work well.

File: islandora_openseadragon.module (hunk 1 above)
Comment (adam-vessey) [inline islandora_openseadragon.module]: Indeed it is... Something about the D7 changes to it makes me usually try to not use it... I'm not sure why... Anyway.  Yeah, I'll make it use it.

**Comment by adam-vessey** [comment]
There we be.

**Pull request merged**